packages/sit-onyx/src/components/OnyxAccordionItem/OnyxAccordionItem.vue

```diff
@@ -0,0 +1,165 @@
+<script lang="ts" setup>
+import chevronLeftSmall from "@sit-onyx/icons/chevron-left-small.svg?raw";
+import { useId } from "vue";
+import { useDensity } from "../../";
+import { SKELETON_INJECTED_SYMBOL, useSkeletonContext } from "../../composables/useSkeletonState";
+import OnyxIcon from "../OnyxIcon/OnyxIcon.vue";
+import OnyxSkeleton from "../OnyxSkeleton/OnyxSkeleton.vue";
+import type { OnyxAccordionItemProps } from "./types";
+import { useAccordionItem } from "./useAccordionItem";
+
+const props = withDefaults(defineProps<OnyxAccordionItemProps>(), {
+  disabled: false,
+  skeleton: SKELETON_INJECTED_SYMBOL,
+});
+
+defineSlots<{
+  /**
+   * Displays the header content.
+   */
+  header(): unknown;
+
+  /**
+   * Displays the panel content.
+   */
+  default?(): unknown;
+}>();
+const itemId = useId();
+
+const { accordionContext, isOpen } = useAccordionItem(itemId);
+
+const skeleton = useSkeletonContext(props);
+const { densityClass } = useDensity(props);
+</script>
+
+<template>
+  <div
+    v-if="skeleton || accordionContext?.skeleton.value"
+    :class="['onyx-component', 'onyx-accordion-item-skeleton', densityClass]"
+  >
+    <OnyxSkeleton class="onyx-accordion-item-skeleton__main" />
+    <OnyxSkeleton class="onyx-accordion-item-skeleton__decoration" />
+  </div>
+
+  <details
+    v-else
+    class="onyx-component onyx-accordion-item"
+    :class="[
+      densityClass,
+      accordionContext?.disabled.value || props.disabled ? 'onyx-accordion-item--disabled' : '',
+    ]"
+    :open="isOpen"
+    @toggle="isOpen = ($event.target as HTMLDetailsElement).open"
+  >
+    <summary
+      :id="'header-' + itemId"
+      class="onyx-accordion-item__header"
+      role="button"
+      :tabindex="accordionContext?.disabled.value || props.disabled ? -1 : 0"
+      :aria-expanded="isOpen"
+      :aria-controls="'panel-' + itemId"
+      :aria-disabled="accordionContext?.disabled.value || props.disabled"
+    >
+      <div class="onyx-accordion-item__header__content">
```

Comment:
In BEM naming only one `__` can be used so we would need to rename this, e.g. to `onyx-accordion-item__header-content`

Reply:
Same for `__header__icon`